config/main.tf

```diff
@@ -57,4 +57,10 @@ resource "google_bigquery_dataset" "bq_cdc_covid_deaths_dataset" {
   location   = "US"
 }
 
+# Create a BigQuery dataset for cdc covid deaths
```

Comment:
Can you instead add this as a separate tf file under config/data_sources (eg add acs_health_insurance.tf in that folder)?

Reply:
Done